docs/.vuepress/tools/jsdoc-convert/index.mjs

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable no-useless-escape */
```

Comment:
```suggestion
```
I think that if the Eslint rule says that there is a useless regex escape character we can remove that. WDYT?